sycl/include/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -17,13 +17,6 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace detail {
 
-// This guard is needed because the libsycl.so can be compiled with C++ <=14
-// while the code requires C++17. This code is not supposed to be used by the
-// libsycl.so so it should not be a problem.
```

Comment:
This comment should probably be moved down to declaration of `get_spec_constant_symbolic_ID`

Reply:
Fixed in b69f889e7241d09db8909931c2277536f97ff019